### ISSUE TITLE
StringQuery does not pass in arguments in both queryList and querySingle

### ISSUE THREAD

**Issue by u007** [agrosner/DBFlow#1420]
DBFlow Version:
Issue Kind (Bug, Question, Feature):
4.0.5

Description: 
queryList in StringQuery only pass in sql, arguments becomes null.
Same goes for querySingle.
This is inconsistent with query() method where it passes in arguments.

Fix is quite simple, passing in cursor instead of query(sql) fixed it.

//sorry this is in kotlin:
```
override fun querySingle(): TModel? {
        val query = getQuery()
        FlowLog.log(FlowLog.Level.V, "querySingle: " + query)
        val cursor = this.query()
        return getSingleModelLoader().load(cursor)
}

override fun queryList(): List<TModel> {
        val query = getQuery()
        FlowLog.log(FlowLog.Level.V, "getQuery: " + query)
        val cursor = this.query()
        return getListModelLoader().load(cursor)
}
```

may need to change for querySingle and queryList that accepts databaseWrapper to use cursor as well.

and also i think it's smelly to be having ListModelLoader extends ModelLoader,
it creates so much confusion that it took me weeks to try to debug the issue (logical error)

hope you understand.

thanks

**Comment by agrosner** [comment]
fixed in 5.0.0-alpha1

**Comment by ANPez** [comment]
A backport for 4.2.4 would be awesome :+1:
